OpenSim/Simulation/Model/BushingForce.cpp

```diff
@@ -200,39 +247,42 @@ double BushingForce::computePotentialEnergy(const SimTK::State& s) const
 //=============================================================================
 // Reporting
 //=============================================================================
-/** 
- * Provide names of the quantities (column labels) of the force value(s) reported
- * 
+/*
+ * Provide names of the quantities (column labels) of the force value(s) to be 
+ * reported.
  */
 OpenSim::Array<std::string> BushingForce::getRecordLabels() const 
 {
-    const string& body1Name = get_body_1();
-    const string& body2Name = get_body_2();
+    const string& frame1Name = getConnectee<PhysicalFrame>("frame1").getName();
+    const string& frame2Name = getConnectee<PhysicalFrame>("frame2").getName();
 
     OpenSim::Array<std::string> labels("");
-    labels.append(getName()+"."+body1Name+".force.X");
-    labels.append(getName()+"."+body1Name+".force.Y");
-    labels.append(getName()+"."+body1Name+".force.Z");
-    labels.append(getName()+"."+body1Name+".torque.X");
-    labels.append(getName()+"."+body1Name+".torque.Y");
-    labels.append(getName()+"."+body1Name+".torque.Z");
-    labels.append(getName()+"."+body2Name+".force.X");
-    labels.append(getName()+"."+body2Name+".force.Y");
-    labels.append(getName()+"."+body2Name+".force.Z");
-    labels.append(getName()+"."+body2Name+".torque.X");
-    labels.append(getName()+"."+body2Name+".torque.Y");
-    labels.append(getName()+"."+body2Name+".torque.Z");
+    labels.append(getName()+"."+frame1Name+".force.X");
+    labels.append(getName()+"."+frame1Name+".force.Y");
+    labels.append(getName()+"."+frame1Name+".force.Z");
+    labels.append(getName()+"."+frame1Name+".torque.X");
+    labels.append(getName()+"."+frame1Name+".torque.Y");
+    labels.append(getName()+"."+frame1Name+".torque.Z");
+    labels.append(getName()+"."+frame2Name+".force.X");
+    labels.append(getName()+"."+frame2Name+".force.Y");
+    labels.append(getName()+"."+frame2Name+".force.Z");
+    labels.append(getName()+"."+frame2Name+".torque.X");
+    labels.append(getName()+"."+frame2Name+".torque.Y");
+    labels.append(getName()+"."+frame2Name+".torque.Z");
```

Comment:
@aseth1 could you add some stuff to the CHANGELOG, including this backwards incompatible change here? Someone could have a postprocessing script that expects "mybushing.pelvis.torque.X"; now they'll get "mybushing.frame1_offset.torque.X",